### ISSUE TITLE
WCAG 2.2 verification

### ISSUE THREAD

**Issue by amyjko** [wordplaydev/wordplay#337]
## What's the maintenance need?

We need as many unit tests, end-to-end tests, and manual tests as possible to verify the entire user interface for WCAG 2.1 compliance. There are likely numerous issues lurking that we cannot see because we're not testing them. This project would involve the creation of many new tests.

## Who benefits?

All creators with disabilities for whom Wordplay is not yet accessibile.

**Comment by amyjko** [comment]
This issue should probably be broken down into smaller issues. I would consider this issue done once there are many other issues open for specific WCAG automated tests that we want to write.

**Comment by amyjko** [comment]
It's the end of Winter! Please provide an update on this issue, including:

* Whether you plan to continue work on it
* If you do not, 1) provide a detailed update of its status, 2) add any unfinished work, 3) list any any unmerged branches, and 4) unassign yourself, so someone else can take over the task. Please leave the issue in better shape than you found it, helping future contributors to start from where you left off.

If you do plan to continue work on it, carry on. Otherwise, thank you for your contributions!

**Comment by amyjko** [comment]
No reply :( Unassigning @MichelleGuan1128 @murbu @meepsterleapster @rf-uw.

**Comment by meepsterleapster** [comment]
Sorry, I ran into major trouble with my computer during the break. More importantly, while I am unlikely to be able to provide the same amount work as last quarter, I would be happy to answer any questions related to end to end tests with playwright that come up, as I spent a significant time using that system for testing for WCAG violations

**Comment by rf-uw** [comment]
Apologies for the late reply - echoing what George said, I am unable to contribute this quarter in the same capacity but will be availible here or on Discord (ryder @ingwion) for questions about e2e accessibility testing with playwright/axe.

**Comment by amyjko** [comment]
Not sure why this was closed, it's not resolved.

**Comment by raghavmalawat** [comment]
Hi @amyjko,

Can you help me in understanding what is the scope of the ticket here. I can help in coming up with a test plan, but would like to understand on what all WCAG 2.2 considerations are our priority.

Also from a product perspective on what are the major flows (features) that needs testing, so that we can narrow down the scope. I have been trying to see how I can come up with a test plan, but seeing the larger scope makes me anxious.

For example
1. Test the sign up / login flow
      - Duplicate email, username, weak password, relevant unit tests (isValidEmail.ts, isValidUsername.ts, tec)
      
Is above fine, or should we also think about Keyboard Navigation, Navigable Buttons, forms, modals, etc. (This will increase the scope drasrtically IMO)

What do you feel about the example above? I would also suggest if you can maybe share an example on how you envision coming up with a test plan, maybe starting with a feature, it will help me in getting clarity (specially because I am still fairly new to the project)

**Comment by amyjko** [comment]
This is more of a meta issue. I would consider it closed once we have issues to capture the need for automated tests for everything that can be automatically tested, and manual tests for anything that can't. That might be hundreds of issues, each of which has its own automated or manual testing work.

@daminihs is working on manual WCAG testing right now, filing issues she finds. She could be a good partner in developing a test plan for this.

**Comment by amyjko** [comment]
I did a thorough WCAG audit recently, so I'm going to close this.
